src/odemis/acq/test/path_test.py

```diff
@@ -1021,5 +1021,91 @@ def test_set_path_stream(self):
         self.assertEqual(self.ccd.fanSpeed.value, 0)
 
 
+class SecomFlimPathTestCase(unittest.TestCase):
+    """
+    Tests to be run with a (simulated) SECOM setup for FLIM
+    """
+    backend_was_running = False
+
+    @classmethod
+    def setUpClass(cls):
+        try:
+            test.start_backend(SECOM_FLIM_CONFIG)
+        except LookupError:
+            logging.info("A running backend is already found, skipping tests")
+            cls.backend_was_running = True
+            return
+        except IOError as exp:
+            logging.error(str(exp))
+            raise
+
+        # Microscope component
+        cls.microscope = model.getComponent(role="secom")
+        # Find CCD & SEM components
+        cls.sft = model.getComponent(role="time-correlator")
+        cls.tc_scanner = model.getComponent(role="tc-scanner")
+        cls.ex_light = model.getComponent(role="light")
+        cls.lscanner = model.getComponent(role="laser-mirror")
+        cls.apd = model.getComponent(role="tc-detector")
+        cls.det0 = model.getComponent(role="photo-detector0")
+        cls.det1 = model.getComponent(role="photo-detector1")
+        cls.det2 = model.getComponent(role="photo-detector2")
+        cls.detsel = model.getComponent(role="det-selector")
+        cls.optmngr = path.OpticalPathManager(cls.microscope)
+
+    @classmethod
+    def tearDownClass(cls):
+        if cls.backend_was_running:
+            return
+        del cls.optmngr  # To garbage collect it
+        test.stop_backend()
+
+    def setUp(self):
+        if self.backend_was_running:
+            self.skipTest("Running backend found")
+
+    def test_set_path_stream(self):
+        self.optmngr.setPath("confocal").result()
+
+        self.optmngr.setPath("flim").result()
+        assert_pos_almost_equal(self.detsel.position.value, {"rx": 1.57}, atol=1e-3)
```

Comment:
This value fails by the way. Apparently it expects 3.14.

Reply:
Indeed... I even wrote it in the YAML file myself. Sorry for the confusion. And apparently I set to just 3.14 (no extra digit), which is probably simpler in some way.